package/cloudshell/orch/training/logic/create_user_sandboxes.py

```diff
@@ -8,7 +8,7 @@
 from cloudshell.orch.training.models.position import Position
 from cloudshell.orch.training.models.training_env import TrainingEnvironmentDataModel
 from cloudshell.orch.training.services.sandbox_components import SandboxComponentsHelperService
-from cloudshell.orch.training.services.email import EmailService
+from cloudshell.email.email import EmailService
```

Comment:
I made a small change to the email package and it allows us to have a nicer import here. 
Check this out: https://github.com/QualiSystemsLab/cloudshell-email/commit/af257dfb624c3d11ae14dd4cfad43827724b687f
Now you can change line 11 to:
from cloudshell.email import EmailService

Reply:
that looks better